src/contracts/tokens/ProtocolToken.sol

```diff
@@ -1,20 +1,27 @@
 // SPDX-License-Identifier: GPL-3.0
 pragma solidity 0.8.20;
 
+import {IProtocolToken} from '@interfaces/tokens/IProtocolToken.sol';
+
 import {ERC20} from '@openzeppelin/contracts/token/ERC20/ERC20.sol';
-import {ERC20Permit} from '@openzeppelin/contracts/token/ERC20/extensions/ERC20Permit.sol';
+import {ERC20Permit, IERC20Permit} from '@openzeppelin/contracts/token/ERC20/extensions/ERC20Permit.sol';
 import {ERC20Votes} from '@openzeppelin/contracts/token/ERC20/extensions/ERC20Votes.sol';
+import {ERC20Pausable, Pausable} from '@openzeppelin/contracts/token/ERC20/extensions/ERC20Pausable.sol';
 import {Time} from '@openzeppelin/contracts/utils/types/Time.sol';
 import {Nonces} from '@openzeppelin/contracts/utils/Nonces.sol';
-import {Authorizable} from '@contracts/utils/Authorizable.sol';
 
-import {IProtocolToken, IERC20Permit} from '@interfaces/tokens/IProtocolToken.sol';
+import {Authorizable} from '@contracts/utils/Authorizable.sol';
 
 /**
  * @title  ProtocolToken
  * @notice This contract represents the protocol ERC20Votes token to be used for governance purposes
  */
-contract ProtocolToken is ERC20, ERC20Permit, ERC20Votes, Authorizable, IProtocolToken {
+contract ProtocolToken is ERC20, ERC20Permit, ERC20Votes, ERC20Pausable, Authorizable, IProtocolToken {
+  // --- Data ---
+
+  /// @inheritdoc IProtocolToken
+  bool public notPausable;
```

Comment:
what? doesn't Pausable already inherits a `paused` var?

Reply:
A `_paused` boolean exists, but cannot be _directly_ overwritten outside `Pausable` because it is `private`.

Reply:
lol, there must be a reason for that 🤔 
that's their design enforcing the dev to use their methods